core/src/com/po/fuck/Assets.java

```diff
@@ -6,22 +6,34 @@
 import com.badlogic.gdx.utils.JsonReader;
 import com.badlogic.gdx.utils.JsonValue;
 
+import java.util.HashMap;
+import java.util.Map;
 
 public class Assets {
-    private static JsonReader jsonReader = new JsonReader();
 
     public static AssetManager manager = new AssetManager();
 
-    public static JsonValue getInfo(String name) {
-        return jsonReader.parse(Gdx.files.internal("assetsData.json")).get(name);
+    private static final String[] basicAssetsNames = new String[]{"laserBeam", "player", "island",
+            "coin", "bullet", "enemy"};
+    private static final String[] weaponAssetsNames = new String[]{"glock", "laserGun"};
+
+    public static Map<String, BasicSpriteInfo> information = new HashMap<>();
+
+    static final JsonValue jsonData = (new JsonReader()).parse(Gdx.files.internal("assetsData.json"));
+
+    public static BasicSpriteInfo getAssetInfo(String name) {
+        return information.get(name);
     }
```

Comment:
This should be split into the loaders, what if I want to load weapon info, what if in the future there will be some other info for some other class ?